connection/aws.go

```diff
@@ -11,6 +11,7 @@ type AWSConnection struct {
 	// ConnectionName of the connection. It'll be used to populate the endpoint, accessKey and secretKey.
 	ConnectionName string       `yaml:"connection,omitempty" json:"connection,omitempty"`
 	AccessKey      types.EnvVar `yaml:"accessKey" json:"accessKey,omitempty"`
+	Bucket         string       `yaml:"bucket,omitempty" json:"bucket,omitempty"`
```

Comment:
Shouldn't we use an S3Connection that extends AWSConnection ?

Reply:
I saw that the existing aws connection struct had some S3 params so I just attached it there. I've added the two new connection structs now.